dist/ui/czi-editor.css

```diff
@@ -104,6 +104,7 @@
 @media only print {
   .ProseMirror[data-layout],
   .ProseMirror .czi-editor-body {
+    margin: 15.5mm !important;
     min-height: 0;
     padding: auto !important;
```

Comment:
I think the right fix is to remove `padding: auto !important;`

Reply:
I just tested removing that and it doesn't fix the problem.